src/main/java/frc/robot/autons/ShootInSpeakerPosition2.java

```diff
@@ -0,0 +1,65 @@
+package frc.robot.autons;
+
+import java.nio.file.Path;
+
+import com.pathplanner.lib.auto.AutoBuilder;
+import com.pathplanner.lib.path.PathPlannerPath;
+
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.ParallelRaceGroup;
+import frc.robot.commands.AcquireCommand;
+import frc.robot.commands.DriveVisionCommand;
+import frc.robot.commands.ScoringPositionCommand;
+import frc.robot.commands.ShootNoteCommand;
+import frc.robot.commands.StorageRunCommand;
+import frc.robot.subsystems.Scoring.ElevatorSetpoints;
+import frc.robot.subsystems.Vision.VisionTarget;
+
+public class ShootInSpeakerPosition2 extends Auton {
+
+    public ShootInSpeakerPosition2(Alliance alliance) {
+        super(alliance);
+
+        PathPlannerPath startToShooting = PathPlannerPath.fromPathFile("From position 2 to speaker");
+        PathPlannerPath shootingToNoteTwo = PathPlannerPath.fromPathFile("From speaker to middle note");
+        PathPlannerPath noteTwoToShooting = PathPlannerPath.fromPathFile("From middle note to speaker");
+        PathPlannerPath shootingToNoteThree = PathPlannerPath.fromPathFile("From speaker to bottom note");
+        PathPlannerPath noteThreeToShooting = PathPlannerPath.fromPathFile("From bottom note to speaker");
+        PathPlannerPath shootingToMidline = PathPlannerPath.fromPathFile("From speaker to midline");
+
+        addCommands(
+                new ParallelRaceGroup(
+                        AutoBuilder.followPath(startToShooting),
+                        new DriveVisionCommand(VisionTarget.APR1)),
```

Comment:
until we get the vision stuff working and testable, could there be a set of autons with vision related commands and a set of autons without

Reply:
I agree, we should hold off on merging any vision autons until they are tested